doc/nrf/known_issues.rst

```diff
@@ -150,14 +151,21 @@ CIA-661: Asset Tracker v2 application configured for LwM2M cannot be built for t
 Serial LTE Modem
 ================
 
-.. rst-class:: v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0
+.. rst-class:: v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0
 
 NCSDK-13895: Build failure for target Thingy:91 with secure_bootloader overlay
   Building the application for Thingy:91 fails if secure_bootloader overlay is included.
 
 Other issues
 ============
 
+.. rst-class:: v2-1-0
+
+NCSDK-15512: Modem traces retrieval incompatible with TF-M
+  In the |NCS| v2.1.0, enabling modem traces prevents TF-M from outputting anything to **UART1**, including output from hardware fault handlers.
+  You can either use **UART1** for TF-M output or for modem traces, but not for both.
+  In earlier |NCS| versions, output from hardware fault handlers from SPM would go to **UART0**, and **UART1** would be used to retrieve modem traces.
```

Comment:
@lemrey , added the known issue. Please check and approve.

Reply:
Affects TF-M since the beginning, should go back to 1.6.0.
Affected Kconfigs are: TFM_LOG_LEVEL_SILENCE (disable UART1 in TF-M), and NRF_MODEM_LIB_TRACE_BACKEND_UART (enable UART1 in modem trace).

Reply:
Looks good

Reply:
Updated.